### ISSUE TITLE
Build issue after import 

### ISSUE THREAD

**Issue by onnimikki** [MohaimenH/K32L2B3-MIDI2#1]
_[not asking you to solve this Mohaimen.  Just making a note to try to 1. figure out if this is the right way to tag issues and 2. to document hiccups.]_

Cloned the GitHub repository locally.  Then imported from filesystem within MCUXpresso.

Import worked.  Trying a build.  Get this error:

_Errors occurred during the build.
Errors running builder 'CDT Builder' on project 'K32L2B31A_Project_TUSB_UMP'.
Cannot invoke "org.eclipse.cdt.managedbuilder.core.IManagedBuildInfo.getDefaultConfiguration()" because "info" is null_

Not sure why.  Will have to check again later.

![Uploading Screenshot 2024-07-09 at 8.32.02 AM.png…]()

**Comment by MohaimenH** [comment]
Did a quick Google search and it seems to be Eclipse project settings related. Can you try deleting the project, downloading the ZIP from release and importing it?

**Comment by onnimikki** [comment]
Yes.  Tried fresh.  This time I used the Eclipse Git tool directly (Projects from Git (with smart import) to clone the repository to a local folder.  ("local destination").  All of that was similar to using the command line clone tool yesterday, but just from the Eclipse GUI.  It then did the import from the local copy ("Import Projects from a File System or Archive")... still from within the eGit tool.  The project that shows up in Eclipse does show, in dark orange text, the name of the local copy of the repository **(in K32L2B3-MIDI2) [K32L2B3-MIDI2 Main]**

<img width="474" alt="Screenshot 2024-07-10 at 6 50 00 AM" src="https://github.com/MohaimenH/K32L2B3-MIDI2/assets/2548792/fca7bdec-bd16-4861-85be-369b07092163">

but build and run are still greyed out.

<img width="476" alt="Screenshot 2024-07-10 at 6 45 26 AM" src="https://github.com/MohaimenH/K32L2B3-MIDI2/assets/2548792/ccc90623-66f5-41cc-80ed-6ff19d0328e8">

**Comment by MohaimenH** [comment]
I am certain it has to do with project settings being configured to my machine...

Can you try without using Git and downloading the release zip? https://github.com/MohaimenH/K32L2B3-MIDI2/releases/tag/v0.1-beta

**Comment by onnimikki** [comment]
Never mind.  It's working now.  I suspected that I simply wasn't selecting the project right.  So I installed an SDK example.  Built it.  Erased and removed it and the other projects that were there, too.  Except for the GitHub clone.  Restart and it defaulted to selecting the GitHub project.  So Build was now available.  Newbie mistake.  

Have connected up the board now, programmed it and now the Mac ID'd it when I connected via the regular USB (not SDA), like this:
<img width="474" alt="Screenshot 2024-07-10 at 7 35 38 AM" src="https://github.com/MohaimenH/K32L2B3-MIDI2/assets/2548792/7fead41f-0e52-40a1-9b55-2ff3ac6e89cc">


That's pretty cool to see.
